dev.json

```diff
@@ -40,6 +40,13 @@
     "intel-nuc": "8.0.dev20220505",
     "khadas-vim3": "8.0.dev20220505"
   },
+  "hassos-upgrade": {
```

Comment:
I dont think this make sense, those on dev/beta we can expect to update more frequently and/or be able to manage this path manually with CLI

Reply:
Hm, what if you just switched to `beta`? It's just a button. E.g. right now, that makes sure that people upgrade to 7.6 before going to 8.x (which they should for `ova`/`generic-x86-64` due to GRUB2 update)/

I :100:  agree for `dev` though, actually better to leave that open.

Reply:
I agree on dev, should always just suggest latest and those users really should be up to date. Plus the versioning scheme is different.

For beta it should be harmless since the last stable release is the last beta release of a major version. And while those users should also be up to date this is an end user selected option and requires manual effort to keep to date. People can forget. Or maybe they switched to beta channel just to beta test core and didn't plan to go to beta versions of OS.

Reply:
> Or maybe they switched to beta channel just to beta test core and didn't plan to go to beta versions of OS.

Indeed, that is probably quite often the case.